packages/analyzer/fixtures/inheritance-superclass/package/implements-element.ts

```diff
@@ -0,0 +1,33 @@
+import { ReactiveController, ReactiveControllerHost } from '@lit/reactive-element';
+
+import { BatchingElement } from './BatchingElement';
+
+export class ImplementsElement extends BatchingElement implements ReactiveControllerHost {
+  classField;
+
+  static get observedAttributes() {
+    return [...super.observedAttributes, 'class-attr']
+  };
+
+  private controllers = new Set<ReactiveController>();
+
+  addController(controller: ReactiveController): void {
+    this.controllers.add(controller);
+  }
+
+  removeController(controller: ReactiveController): void {
+    this.controllers.add(controller);
+  }
+
+  async requestUpdate(): Promise<boolean> {
+    return Promise.resolve(true);
+  }
+
+  get updateComplete(): Promise<boolean> {
+    return this.requestUpdate();
+  }
```

Comment:
Could we remove these methods, or change them for 1 `implementsMethod()`? I'd like to keep the fixtures as concise as possible so its easy in the future to see whats going on vs whats expected

Reply:
doesn't bother me, but those are the interface specified

Reply:
done